spec/models/hardware_spec.rb

```diff
@@ -196,28 +196,28 @@
     end
   end
 
-  describe ".ram_size_in_bytes" do
```

Comment:
you just changed this spec. and the new one looks great.

Can you copy this change, revert it, paste it?

so we end up with the original spec and a basically duplicate one?

In the memory_mb one (the original), please add a check that ramsize has the right value when setting memory mb in the factory